rb/utils.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+import sys
+
+PY2 = sys.version_info[0] == 2
+
+if PY2:
+    integer_types = (int, long)
+
+    text_type = unicode
+
+    bytes_type = str
+
+    _iteritems = "iteritems"
+    _itervalues = "itervalues"
+
+    from itertools import izip
+else:
+    integer_types = (int,)
+
+    text_type = str
+
+    bytes_type = bytes
+
+    izip = zip
+
+    _iteritems = "items"
+    _itervalues = "values"
+
+
+def iteritems(d, **kw):
+    return iter(getattr(d, _iteritems)(**kw))
+
+
+def itervalues(d, **kw):
+    return iter(getattr(d, _itervalues)(**kw))
```

Comment:
The following would probably be more performant (if these are called frequently in hot spots):

```py
if PY2:
    ...
    def iteritems(d, **kw):
        return iter(d.iteritems(**kw))
    
    def itervalues(d, **kw):
        return iter(d.itervalues(**kw))
else:
    ...
    def iteritems(d, **kw):
        return iter(d.items(**kw))
    
    def itervalues(d, **kw):
        return iter(d.values(**kw))
```

Reply:
Yeah good call, will swap it over to that

Reply:
https://github.com/getsentry/rb/pull/43